### PR TITLE
Enable dual sim on F5122

### PR DESCRIPTION
**Pull request by jonnius**

### CODE REVIEW

File: rootdir/system/etc/unblock_wakelock.sh [hunk 1]
```diff
@@ -1,14 +1,14 @@
 #!/system/bin/sh
 
-#wakelock=sns_periodic_wakelock
+wakelock=sns_periodic_wakelock
```
Comment (jonnius) [inline rootdir/system/etc/unblock_wakelock.sh]: Did I open the PR against the wrong branch?

Comment (fredldotme) [reply rootdir/system/etc/unblock_wakelock.sh]: The branch is correct, wondering if you branched off an older state of the repo.

Comment (jonnius) [reply rootdir/system/etc/unblock_wakelock.sh]: Fixed it.

**Review by fredldotme — APPROVED** [review]
LGTM

**Pull request merged**